features/admin.users.v1/components/user-profile/fields/form-field-renderer.tsx

```diff
@@ -507,7 +507,7 @@ const ProfileFormFieldRenderer: FunctionComponent<ProfileFormFieldRendererPropsI
             );
 
         case ClaimInputFormat.DATE_PICKER:{
-            const formattedInitialValue: moment.Moment = moment(initialValue, "YYYY-MM-DD", true);
+            const formattedInitialValue: dayjs.Dayjs = dayjs(initialValue as string, "YYYY-MM-DD", true);
```

Comment:
shall we directly import `Dayjs` from the `dayjs` package instead? that'd help us reduce the bundle size further.